src/net/sourceforge/kolmafia/session/LoginManager.java

```diff
@@ -96,14 +97,14 @@ private static boolean ping(int attempt) {
         // The user wants to be "close" to the best ping seen.
         // Get the shortest ping test time we've seen. If the ping test we
         // just ran is the first, that will be it.
-        double threshold = 1.0 + Preferences.getFloat("pingLoginGood");
+        double threshold = 1.0 + Preferences.getDouble("pingLoginThreshold");
```

Comment:
Change from pingLoginGood to pingLoginThreshold is deliberate, I presume.

Reply:
Yes. The GUI has always configured pingLoginThreshold and that is what is in defaults.txt.